packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -33,8 +33,10 @@ import 'theme.dart';
 
 typedef _FlexibleConfigBuilder = _ScrollUnderFlexibleConfig Function(BuildContext);
 
-const double _kLeadingWidth = kToolbarHeight; // So the leading button is square.
-const double _kMaxTitleTextScaleFactor = 1.34; // TODO(perc): Add link to Material spec when available, https://github.com/flutter/flutter/issues/58769.
+const double _kLeadingWidth =
```

Comment:
This file and the next one still have their formatting all messed up. Please revert all formatting changes back to the original.